src/zcl_abapgit_tadir.clas.abap

```diff
@@ -98,7 +99,13 @@ CLASS zcl_abapgit_tadir IMPLEMENTATION.
     ENDIF.
 
     IF NOT io_dot IS INITIAL.
-      lv_path = zcl_abapgit_folder_logic=>package_to_path(
+      "Reuse given Folder Logic Instance
+      IF lo_folder_logic IS NOT BOUND.
```

Comment:
this is always initial

will the optimization of TADIR class be part of this PR, or https://github.com/larshp/abapGit/pull/1725 ?